app/templates/src/assets/scss/_theme.scss

```diff
@@ -0,0 +1,23 @@
+// Override and extend Bootstrap stuff
+// --------------------------------------------------
+// Files, classes, mixins etc.
+<% if (templateProps.boilerplateAmount === 'Just a little – Get started with a few example files') { %>
+@import "<%= templateProps.theme %>/mixins";
+@import "<%= templateProps.theme %>/scaffolding";
+@import "<%= templateProps.theme %>/alerts";
+<% } %>
+
+// Own modules
+// --------------------------------------------------
+// @import "theme/testResponsiveHelpers"; // debug
+<% if (templateProps.boilerplateAmount === 'Just a little – Get started with a few example files') { %>
+@import "<%= templateProps.theme %>/demoElements";
+@import "<%= templateProps.theme %>/footer";
+<% } %>
```

Comment:
This is causing the stylelint errors.
It produces:

```
// Override and extend Bootstrap stuff
// --------------------------------------------------
// Files, classes, mixins etc.


// Own modules
// --------------------------------------------------
// @import "theme/testResponsiveHelpers"; // debug


// Important note //
// You could also use this file to insert theme related style definitions
// directly within this file. But we recommend to exclude your LESS code to
// separate files like the examples above when you exceed a few hundred lines
// of code. Otherwise it will definitely have a negative impact on
// maintainability.

```

Compare https://github.com/micromata/generator-bootstrap-kickstart/blob/master/app/templates/src/assets/less/_theme.less to https://github.com/micromata/generator-bootstrap-kickstart/blob/4.0-dev/app/templates/src/assets/scss/_theme.scss